test/controllers/api/files_controller_test.rb

```diff
@@ -53,6 +53,39 @@ class API::FilesControllerTest < API::BaseTestCase
     assert content, response.body
   end
 
+  test "show should return special README.md solution file" do
+    setup_user
+    solution = create :concept_solution, user: @current_user
+    create :submission, solution: solution
+    content = "README"
+
+    get "/api/v1/solutions/#{solution.uuid}/files/README.md", headers: @headers, as: :json
+    assert_response 200
+    assert content, response.body
```

Comment:
I copied this from the other tests, but is this not actually a bad test? It doesn't fail where I think it should be. It looks like the original intent might have been to check if the body contained a certain text.

Reply:
>  It doesn't fail where I think it should be.

Where is it failing and where do you think it should?

I can't see what's wrong with it, but I may well be just missing something.

Reply:
What does `assert content, response.body` do? If I change the content to any random string, it still passes the test. I assume that we want to do something like `assert_includes response.body, content`. I've updated the tests.

Reply:
Ah. I see. Yes. Or `assert_equal` probably.